softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -38,6 +40,41 @@ function activityMarkdown(activity, language) {
   return markdown;
 }
 
+function createChunkedSendingReportMessage(statusmessages, subject, sender) {
+  const anySendingError = R.any(statusmessage.isErrorMessage, statusmessages);
+  const resultMessage = new Message(
+    {
+      subject,
+      markdown: anySendingError
+        ? `Fehler: ${statusmessages.map((s) => s.contents().additionalArguments.err).join(" ")}`
+        : "E-Mails erfolgreich versendet",
+    },
+    sender,
+  );
+  resultMessage.setTo(sender.email());
+  return resultMessage;
+}
+
+function sendMailInChunks(maxMailSendingChunkSize, allMembers, message, type, sender) {
+  const membersInChunks = R.splitEvery(maxMailSendingChunkSize, allMembers);
+
+  Promise.all(
+    membersInChunks.map((bccs) => {
+      message.setBccToMemberAddresses(bccs);
+      return sendMail(message, type);
+    }),
+  )
+    .then((statusmessages) => {
+      const resultMessage = createChunkedSendingReportMessage(statusmessages, `Report "${message.subject}"`, sender);
+      sendMail(resultMessage, type);
+    })
+    .catch((err) => {
+      logger.error(err);
+    });
+
+  return mailtransport.statusmessageForSuccess();
```

Comment:
DAs kommt zurück, bevor die Statusnachricht rausgeht.

Reply:
War auch mein Plan: da ja der Versand der ganzen Mails zu lange dauert, dachte ich, am besten das ganze im Hintergrund machen und schonmal vom express-Handler ein Ok rausgeben, damit man sicher kein Timeout bekommt. Ist das problematisch?

Reply:
Oder anders gefragt. Welche der Umsetzungsoptionen soll ich machen:
* wenn man mehr als die max. Chunk-Size (deduplizierte) BCCs hat, dann wird an die UI gleich ein: ist unterwegs zurückgegeben, auch wenn die Mails noch verschickt werden (also den Ansatz den ich hier gemacht habe). Wenn das alles durch ist, dann wird eine Mail an den Absender verschickt, mit dem Versende-Report.
* oder man verschickt doch alle Mails (aber eben in mehrere Chunks aufgeteilt) und erst dann gibt man ein Ergebnis. Da man dann aber auch weiß ob alles geklappt hat, braucht man dann überhaupt den Versende-Report per Mail?

Reply:
Am liebsten wäre mir: warten, bis alles raus sind

Reply:
Ich befürchte halt, dass wir dann auch wieder Timeouts bekommen bzw. manche der Mails nicht rausgehen (die nur teilweise Zustellung ist ja gerade die Motivation um das umzubauen).